guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -209,6 +210,22 @@ Sets the current screen's available orientations and forces rotation if current
 
 Defaults to `default`.
 
+### `searchBar` (iOS only)
```

Comment:
There is no such prop in `ScreenStackHeaderConfig`. Please add the correct info about adding a child to `ScreenStackHeaderSearchBarView`, maybe with an example.

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/758/commits/49247f00af70303e425be0695dba328f3e2b4d4a